src/main/java/ch/njol/skript/events/EvtClick.java

```diff
@@ -31,43 +32,40 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.PlayerInventory;
 import org.eclipse.jdt.annotation.Nullable;
+import org.skriptlang.skript.lang.comparator.Relation;
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.bukkitutil.ClickEventTracker;
-import org.skriptlang.skript.lang.comparator.Relation;
 import ch.njol.skript.classes.data.DefaultComparators;
 import ch.njol.skript.entity.EntityData;
 import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptEvent;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.skript.log.ErrorQuality;
 import ch.njol.util.Checker;
 import ch.njol.util.coll.CollectionUtils;
 
-@SuppressWarnings("unchecked")
 public class EvtClick extends SkriptEvent {
 
 	/**
 	 * Click types.
 	 */
 	private final static int RIGHT = 1, LEFT = 2, ANY = RIGHT | LEFT;
```

Comment:
```suggestion
	private enum ClickType {
		RIGHT,
		LEFT,
		ANY
	}
```

Reply:
Then use a field, `ordinal()` or `name().toLowerCase()`

Reply:
Would be nice! but it's out-of-scope for now, maybe in another PR \:)